### DISCUSSION TITLE
怎么克隆仓库到本地

### DISCUSSION THREAD

**Discussion by kuanghy123** [OpenHUTB/hutb#118] [Q&A]
怎么克隆仓库到本地

**Comment by ZLW523523** [comment]
在新文件夹中git clone你自己的网址

**Comment by XieTJ** [comment]
补充一下，克隆前要先确保本地装了 Git，而且如果仓库比较大，用 git clone --depth=1 只克隆最新版本会快很多，避免网络超时。我之前克隆时遇到过权限问题，提醒大家如果用 HTTPS 地址要输入 GitHub 账号密码，用 SSH 地址会更方便，提前配置好密钥就行。
